.gitignore

```diff
@@ -0,0 +1 @@
+bower_components
```

Comment:
Don't forget to `bower init`. That creates your `bower.json` file that helps devs who pull down your code run your project the way it was meant to be run without having to dig to find which dependencies you are using.